keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -292,29 +546,245 @@ N/A
 
 N/A
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+It's up to the user what package repository (OBS or Google) they want to use.
+In case OBS doesn't work for them, they can reconfigure their systems to use
+the Google package repository.
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+N/A. This is configured on the operating system (i.e. package manager) level. 
+
+###### Does enabling the feature change any default behavior?
+
+Not anticipated. We're trying to match the existing spec files as best as we
+can.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. Users can rollback to the Google package repository.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+There are no side effects anticipated.
+
+###### Are there any tests for feature enablement/disablement?
+
+N/A
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+N/A
+
+###### What specific metrics should inform a rollback?
+
+Installation and upgrading issues. For example, if a package upgrade is not
```

Comment:
Would we only have users roll back due to technical issues? Is there no user-based or engineering-based criteria that we’d decide to not go with OBS after this implementation?

Reply:
> Is there no user-based or engineering-based criteria that we’d decide to not go with OBS after this implementation?

I don't really see user-based or engineering-based criteria to prefer Google registries over OBS. We build packages in a very similar way so that the migration is as seamless as possible.

Reply:
I was thinking more if we run into any technical blockers for using OBS or issues with integrating OBS with our release process. Put another way, is there anything that would require us to reevaluate our approach to OBS or do we feel like we've tested it enough that it could handle everything we throw at it?

I assumed this area of a KEP, in a traditional sense, should address the "What metrics should inform RelEng to rollback? (the KEP)" vs. "What metrics should inform an end user?"

I can live with "we are running side-by-side so there's no need to rollback" but if we approach it this way, do we have the criteria written for how/when we sunset the Google registries based on X success metrics?